6.0-alpine/run.sh

```diff
@@ -6,8 +6,7 @@ if [ "${1:0:1}" != '-' ]; then
   exec "$@"
 fi
 
-exec java -jar lib/sonar-application-$SONAR_VERSION.jar \
-  ${SONARQUBE_JAVA_ARGS} \
+exec java ${SONARQUBE_JAVA_ARGS} -jar lib/sonar-application-$SONAR_VERSION.jar \
```

Comment:
I fear that you might be correct :-(

How should those be set, since I can't modify the config file at the moment?